HISTORY.md

```diff
@@ -1,3 +1,16 @@
+4.1.0 / 2019-11-04
```

Comment:
This should just be the word "unreleased" and have the `=====` separator below it. The version number and date will get populated on publish.